GPflowOpt/acquisition/acquisition.py

```diff
@@ -56,7 +86,7 @@ def __init__(self, models=[], optimize_restarts=5):
 
         assert (optimize_restarts >= 0)
         self.optimize_restarts = optimize_restarts
-        self._optimize_models()
+        self._needs_setup = True
```

Comment:
this happens in the constructor, under this setting highest_parent == self because its not in a hierarchy yet.